<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>423</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="UsernameLabel">
      <property name="text">
       <string>Username:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPlainTextEdit" name="InputUsername">
      <property name="plainText">
       <string>User1</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="IPLabel">
      <property name="text">
       <string>IP:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPlainTextEdit" name="InputIP">
      <property name="plainText">
       <string>127.0.0.1</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="PortLabel">
      <property name="text">
       <string>Port:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPlainTextEdit" name="InputPort">
      <property name="plainText">
       <string>1234</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="ConnectButton">
      <property name="text">
       <string>Connect</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="DestUsernameLabel">
      <property name="text">
       <string>Dest Username:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="ComboBoxDestUsername">
      <item>
       <property name="text">
        <string>Choose a Username from this list</string>
       </property>
      </item>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>31</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuBonjour">
    <property name="title">
     <string>Babe&amp;l Client</string>
    </property>
   </widget>
   <addaction name="menuBonjour"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
